### ISSUE TITLE
A11y_ML .NET Model Builder_ Scenarios_ Screen Reader: Narrator/NVDA is not reading any information when focus comes to next steps controls

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1597]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #WCAG1.3.1; #ScreenReader; #Narrator; #NVDA; #A11yPreview;

### Environment Details:
Microsoft Visual Studio

### Repro Steps:

1. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#)
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Text Classification" and press enter
5. Select "Local (CPU)" environment and tab till "Next" button and press enter
6. In "Data Source Type" and select "File" radio button
7. Tab till "Browse" button and select the input file
8. Tab till "Column to predict" control and select a value and then tab till "Next" button and press enter
9. Tab till "Start Training" button and press enter
10. Once training is completed tab till "Next" button and press enter
11. In "Evaluate" tab navigate till "Next" button and press enter
12. In consume tab navigate till "Next" button and press enter
13. In "Next Steps" tab verify whether screen readers are reading the information about the controls properly or not

### Actual Result:
Narrator/NVDA is not reading any information when focus comes to next steps controls

### Expected Result:
Narrator/NVDA should read the information about the controls properly

### User Impact:
Screen Reader users will not be able to understand about the controls if screen reader doesn't read any information about the controls

**Comment by beccamc** [comment]
Validate on  rel/candidate build 16.7.2.2141903. See issue 1627 for instructions for install.

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. Narrator/NVDA is announcing about Next steps controls when focus lands on the controls.

https://user-images.githubusercontent.com/83570638/131455818-56ed9308-31b4-48bb-b2f3-3a89e9102003.mp4

![1597_NVDA_fixed](https://user-images.githubusercontent.com/83570638/131455831-1a55c0a7-1b04-4e1f-a5fb-fe9d81ad360e.png)

**Comment by akashelijah05** [comment]
#Closed;
